scripts/partition_manager/partition_manager.rst

```diff
@@ -96,34 +99,83 @@ placement: dict
          Place the partition after the first existing partition in the list.
 
      Valid values in the lists are ``app``, ``start``, ``end``, or the name of any partition.
+     The value ``start`` refers to the flash device's start address.
+     The value ``end`` refers to its end address.
      It is not possible to place the partition after ``end`` or before ``start``.
 
       align: dict
          Ensure alignment of start or end of partition by specifying a dict with a ``start`` or ``end`` key respectively, where the value is the number of bytes to align to.
          If necessary, empty partitions are inserted in front of or behind the partition to ensure that the alignment is correct.
          Only one key can be specified.
-         Partitions which directly or indirectly (through spans) share size with the ``app`` partitions can only be aligned if they are placed directly after the ``app`` partition.
+         Partitions which directly or indirectly (through :ref:`spans <partition_manager_spans>`) share size with the ``app`` partitions can only be aligned if they are placed directly after the ``app`` partition.
+
+   An empty value is also permitted, like so:
+
+   .. code-block:: yaml
+
+      mcuboot:
```

Comment:
Note this is copy/pasted from the span examples below, so if something about this example is incorrect, then the examples also seem to be incorrect and should be fixed.